index.html

```diff
@@ -3953,64 +3953,40 @@ <h3>JSON-LD</h3>
         <section class="informative">
           <h4>Data Model Foundations</h4>
           <p>
-  The world of credentials is vast and varied. A physical wallet places
-  relatively few constraints on the shape and size of those credentials — and it
-  places no restriction on their contents. Likewise, the Verifiable Credential Data
-  Model's use of [[!JSON-LD]] provides the consistent shape for all compliant
-  Verifiable Credentials, so that digital wallets can store credentials with
-  widely varying content, providing value to a plethora of communities, and
-  enabling many diverse use cases — from coupons to authentication.
+  The world of credentials is vast and varied. A physical wallet places a few
+  constraints on the shape and size of those credentials, and no restriction on
+  their contents. Likewise, the Verifiable Credential Data Model's use of
+  [[!JSON-LD]] provides a consistent way to interpret all compliant Verifiable
+  Credentials using a syntax that can be enforced using complementary
+  technologies such as JSON Schema. This means issuers can add claim terminology
+  unique to their communities without the high costs of centralized terminology
+  registration or format churn. In turn, digital wallets can index and store
```

Comment:
Indexing doesn't happen without storage, so storage should be mentioned first.
```suggestion
  registration or format churn. In turn, digital wallets can store and index
```